src/database/DataTypes/TransactionBatch.js

```diff
@@ -146,14 +143,18 @@ export class TransactionBatch extends Realm.Object {
   toString() {
     return `${this?.itemBatch} in a ${this.transaction.type}`;
   }
+
+  get otherPartyName() {
+    return `${this.transaction?.otherParty?.name}`;
```

Comment:
Nit: just wondering if it might be better to use an empty string rather than stringify `"null"` here?

Reply:
yeah 1133d8b9b950e87191337874fe393ffa7efad04f